sdk/search/Azure.Search.Documents/src/Azure.Search.Documents.csproj

```diff
@@ -41,4 +41,7 @@
     <PackageReference Include="System.Text.Json" />
     <PackageReference Include="System.Threading.Channels" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj" />
```

Comment:
This should be conditioned on some property we can set (or otherwise unset). See sdk\search\Azure.Search.Documents\Directory.Build.props for examples. This makes it easy to condition in (or out) as needed if a servicing release had to be done, or it took longer than expected for this experimental feature. We did this with `FieldBuilder` support for spatial types and JSON object serializer (or something like that).

Reply:
Thanks for the hint. I am removing the reference from the project file.